### PR TITLE
Ayush's tests

### PR DESCRIPTION
**Pull request by akohli96**
Dawgs, I basically just added my tests.
There is not much to review but I wanted to stick to good coding practices so asking for review

### CODE REVIEW

File: mhapsite/mhap/views.py [hunk 1]
```diff
@@ -205,20 +205,21 @@ def index(request):
...
+    
     context = {
         "data": data,
         "data_slugs": data_slugs,
         "user_prof": user_prof,
         "instance": instance,
         #first variable is what is referenced in html
         #second variable is in code
-        "quote_text":second_quote.quote,
+        "quote_text":second_quote.quote
```
Comment (perpetual-pelican) [inline mhapsite/mhap/views.py]: Correct me if I'm wrong, but don't you need the comma here? Is there a particular reason you removed the comma after quote_text?

Comment (akohli96) [reply mhapsite/mhap/views.py]: Good catch

Comment (akohli96) [reply mhapsite/mhap/views.py]: Comma added James in new commit

**Review by perpetual-pelican — APPROVED** [review]
Okay, the comma thing was my only issue. Should be all good now.

File: mhapsite/mhap/views.py [hunk 2]
```diff
@@ -205,12 +205,13 @@ def index(request):
 
 
 
-    #quote=None
+    quote=None
     try:
         second_quote = Quote.objects.get(id=2)
+        quote=second_quote
```
Comment (markiragoldberg) [inline mhapsite/mhap/views.py:211]: Everything looks syntactically correct but I don't understand the purpose of these changes in views.py.

Comment (akohli96) [reply mhapsite/mhap/views.py:211]: I was thinking of backup if we cant fetch second quote

**Pull request merged**